src/common/fanctl.h

```diff
@@ -65,9 +70,11 @@ class CFanCtlPWM : private fanctl_pwm_t {
     inline uint8_t get_min_PWM() { return min_value; }
     inline uint8_t get_max_PWM() { return max_value; }
     inline uint8_t get_PWM() { return pwm; }
```

Comment:
const ... and the other methods as well...

Reply:
fixed